structure_tests/ext_run.sh

```diff
@@ -100,13 +100,12 @@ while test $# -gt 0; do
 			if test $# -eq 0; then
 				usage
 			else
-				FULLPATH=$(readlink -f "$1")
-				if [ ! -d "$FULLPATH" ]; then
-					echo "$FULLPATH is not a valid directory."
-					cleanup
-					exit 1
+				if [ ! -d "$1" -o ! -d $(readlink -f "$1") ]; then
```

Comment:
looks like shellcheck wants you to use `||` instead of `-o`: https://travis-ci.org/GoogleCloudPlatform/runtimes-common/builds/261877654